### DISCUSSION TITLE
Adjustable Artifact Expiration

### DISCUSSION THREAD

**Discussion by chingc** [community/community#25458] [Actions]
<p>This doc says artifacts automatically expire in 90 days.</p>
<p><a href="https://help.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts#downloading-or-deleting-artifacts" rel="nofollow noopener">https://help.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts#downloading-or-deleting-artifacts</a></p>
<p>Is it possible to adjust this number somewhere?  If not, I think it makes for a nice feature.</p>

**Comment by AndreaGriffiths11** [comment]
<p>HI <a class="mention" href="/chingc">@chingc</a>,</p>
<p>Thank you for being here! There is not currently a way to configure custom expiration dates for artifacts.</p>
<p>Thanks for the feedback! I’ve taken your suggestion and passed it along to the appropriate teams. Thanks again for reaching out :grinning:</p>

**Comment by chingc** [comment]
<p><a class="mention" href="/andreagriffiths11">@andreagriffiths11</a> Hi, I was just wondering if there’s any word as to whether there are plans to implement this.</p>

**Comment by iBug** [comment]
<p>I’d love to see this implemented. Even if we can’t set the expiration time to an arbitrary value, providing options for “tiers” like 90d (current) / 30d / 7d would be nice and way better than the status quo.</p>

**Comment by ahdbilal** [comment, marked as answer]
<p>You can now change artifact and log retention duration both from Actions Settings tab or via YAML. You can learn more about this feature <a href="https://github.blog/changelog/2020-10-08-github-actions-ability-to-change-retention-days-for-artifacts-and-logs/">here</a>. Feedback is welcome!</p>
